.gitignore

```diff
@@ -5,7 +5,6 @@
 dist/
 target/
 lib_managed/
-src_managed/
```

Comment:
> I know that the src_managed folder is inside the target folder which is already ignored...

Yes exactly, it's unnecessary to list it since it's already covered by target.

Reply:
I'm not sure either...